diff -burNp motion-3.2.11/Makefile.in motion-3.2.11-new/Makefile.in
--- motion-3.2.11/Makefile.in	2008-09-22 01:20:58.000000000 +0200
+++ motion-3.2.11-new/Makefile.in	2008-10-05 20:56:06.874207953 +0200
@@ -30,7 +30,7 @@ examplesdir = $(datadir)/@PACKAGE_NAME@-
 # These variables contain compiler flags, object files to build and files to   #
 # install.                                                                     #
 ################################################################################
-CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"@PACKAGE_VERSION@\" -Dsysconfdir=\"$(sysconfdir)\" 
+CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"@PACKAGE_VERSION@\" -Dsysconfdir=\"$(sysconfdir)/motion\" 
 LDFLAGS      = @LDFLAGS@
 LIBS         = @LIBS@ 
 VIDEO_OBJ    = @VIDEO@
