--- motion-3.2.10/Makefile.in.orig	2008-06-10 19:01:53.000000000 +0200
+++ motion-3.2.10/Makefile.in	2008-06-10 19:01:57.000000000 +0200
@@ -31,7 +31,7 @@
 # install.                                                                     #
 ################################################################################
 CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"@PACKAGE_VERSION@\"  \
-	       -Dsysconfdir=\"$(sysconfdir)\" -Ddocdir=\"$(docdir)\"
+	       -Dsysconfdir=\"$(sysconfdir)\motion\" -Ddocdir=\"$(docdir)\motion\"
 LDFLAGS      = @LDFLAGS@
 LIBS         = @LIBS@ 
 VIDEO_OBJ    = @VIDEO@
