diff -Nur motion-3.2.7.orig/Makefile.in motion-3.2.7/Makefile.in
--- motion-3.2.7.orig/Makefile.in	2006-10-20 00:21:54.000000000 +0000
+++ motion-3.2.7/Makefile.in	2007-03-02 08:51:17.000000000 +0000
@@ -31,7 +31,7 @@
 # install.                                                                     #
 ################################################################################
 CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"$(VERSION)\" -D_REENTRANT \
-	       -Dsysconfdir=\"$(sysconfdir)\"
+	       -Dsysconfdir=\"$(sysconfdir)/motion\"
 LDFLAGS      = @LDFLAGS@
 LIBS         = @LIBS@ 
 VIDEO_OBJ    = @VIDEO@
diff -Nur motion-3.2.7.orig/motion-dist.conf motion-3.2.7/motion-dist.conf
--- motion-3.2.7.orig/motion-dist.conf	2006-10-20 00:21:54.000000000 +0000
+++ motion-3.2.7/motion-dist.conf	2007-03-02 08:53:12.000000000 +0000
@@ -11,7 +11,7 @@
 daemon on
 
 # File to store the process ID, also called pid file. (default: not defined)
-process_id_file /var/run/motion.pid 
+process_id_file /var/run/motion/motion.pid 
 
 ############################################################
 # Basic Setup Mode
@@ -323,7 +323,7 @@
 
 # Target base directory for pictures and films
 # Recommended to use absolute path. (Default: current working directory)
-target_dir /usr/local/apache2/htdocs/cam1
+target_dir /home/services/httpd/html/motion/
 
 # File path for snapshots (jpeg or ppm) relative to target_dir
 # Default: %v-%Y%m%d%H%M%S-snapshot
