--- motion-3.2.10/Makefile.in.orig.inttypes	2008-05-18 16:58:52.000000000 +0200
+++ motion-3.2.10/Makefile.in	2008-06-10 19:11:56.000000000 +0200
@@ -31,7 +31,7 @@
 # install.                                                                     #
 ################################################################################
 CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"@PACKAGE_VERSION@\"  \
-	       -Dsysconfdir=\"$(sysconfdir)\motion\" -Ddocdir=\"$(docdir)\motion\"
+	       -Dsysconfdir=\"$(sysconfdir)\motion\" -Ddocdir=\"$(docdir)\motion\" @DEFS@
 LDFLAGS      = @LDFLAGS@
 LIBS         = @LIBS@ 
 VIDEO_OBJ    = @VIDEO@
