diff -urN motion-3.0.6.org/configure.in motion-3.0.6/configure.in
--- motion-3.0.6.org/configure.in	2003-06-10 10:35:18.000000000 +0200
+++ motion-3.0.6/configure.in	2003-06-10 10:40:46.000000000 +0200
@@ -43,12 +43,11 @@
   dnl AUTODETECT ??? SHARED LIB ?????
   AC_MSG_RESULT(not specified)
 else
-  if test -f ${LIBAVCODEC}/libavcodec.a; then
+  if test -f ${LIBAVCODEC}/libavcodec.so; then
     AC_MSG_RESULT(found)
-    CFLAGS="-I${LIBAVCODEC} ${CFLAGS}"
+    CFLAGS="-I/usr/include/ffmpeg ${CFLAGS}"
     CFLAGS="${CFLAGS} -DHAVE_FFMPEG"
-#    LIBS="$LIBS -L${LIBAVCODEC} -lavcodec -lm"
-    LIBS="$LIBS ${LIBAVCODEC}/libavcodec.a -lm"
+    LIBS="$LIBS -L${LIBAVCODEC} -lavcodec -lm"
   else
     AC_MSG_RESULT(not found)
   fi

