--- motion-3.2.6.orig/Makefile.in	2006-05-14 20:30:22.000000000 +0200
+++ motion-3.2.6/Makefile.in	2006-05-14 20:40:27.000000000 +0200
@@ -31,7 +31,7 @@
 # install.                                                                     #
 ################################################################################
 CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"$(VERSION)\" -D_REENTRANT \
-	       -Dsysconfdir=\"$(sysconfdir)\"
+	       -Dsysconfdir=\"$(sysconfdir)/motion\"
 LDFLAGS      = @LDFLAGS@
 LIBS         = @LIBS@ 
 VIDEO_OBJ    = @VIDEO@
